plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/OpenShiftPreferencePage.java

```diff
@@ -128,6 +128,9 @@ private boolean validateLocation(String location) {
 			cliLocationEditor.setErrorMessage(NLS.bind("{0} does not have execute permissions.", file));
 			return false;
 		}
+		if (!new OCBinaryValidator(location).isCompatibleForPublishing()) {
+	        setMessage("Your Openshift 3 client executable is pre-1.1.1 and may cause rsync problems", WARNING);
```

Comment:
@maxandersen I don't grok your comment

I'd just say something like "OpenShift client version 1.1.1 or higher is required to avoid rsync issues"